### DISCUSSION TITLE
Destroyed layout in WSL2?

### DISCUSSION THREAD

**Discussion by K2ouMais** [JanDeDobbeleer/oh-my-posh#1511] [General]
Hello everyone,

I installed oh-my-posh in WSL2 and it seems to work fine.

The only problem I am having is when I first start my Windows Terminal the first time after closing it.

It looks like this:
![{BA81B22B-201F-46E1-B0F6-66DEFCADA76C}](https://user-images.githubusercontent.com/52510455/147583181-917c2bb6-3558-4f1e-840b-d1223b420dcf.png)

After I type "clear" everything looks normal again.

Do you have an idea what could be happening?

Thanks in advance

**Comment by JanDeDobbeleer** [comment]
That's a Windows terminal bug. No fix is known :-)

**Comment by K2ouMais** [comment]
I found a way to not have it anymore.

I just put the command "clear" as the last line on my .bashrc.

Problem solved 👍🏻

**Reply by JanDeDobbeleer** [reply]
@K2ouMais yes, that's the workaround indeed :-)
